x-pack/plugins/fleet/common/openapi/paths/package_policies.yaml

```diff
@@ -44,15 +44,18 @@ post:
     content:
       application/json:
         schema:
-          allOf:
-            - $ref: ../components/schemas/new_package_policy.yaml
-            - type: object
-              properties:
-                id:
-                  type: string
-            - type: object
-              properties:
-                force:
-                  type: boolean
+          oneOf:
+            - $ref: ../components/schemas/simplified_package_policy.yaml
+            - deprecated: true
```

Comment:
Is the deprecated flag at the right place? It is not very clear if it refers to the `allOf` part after or the whole `oneOf`

Reply:
yes it's in the good place it refer to the `allOf` I am going to add a comment to clarify that